tests/software_tests/database/data_record/test_mapping_data_record.py

```
@@ -1,82 +1,540 @@
+
+    @pytest.mark.parametrize("dtc_status_values, expected_output", [
+        ([0x00], {
+            "name": "DTC Status",
+            "physical_value": ("Inactive",),
+            "children": [
+                (
+                        {
+                            "name": "Warning Indicator Requested/MIL on",
+                            "raw_value": 0,
+                            "physical_value": "No",
+                            "children": tuple(),
```

Ruff (0.12.2)

Unnecessary `tuple()` call (rewrite as a literal)

Rewrite as a literal

(C408)